cmd/dalc/main.go

```diff
@@ -61,31 +65,42 @@ func initCmd() *cobra.Command {
 				return err
 			}
 
+			hm := server.HeightMapper{}
+			err = hm.SaveToFile(path + "height_map.json")
+			if err != nil {
+				return err
+			}
+
+			fmt.Println("Please add a key to the keyring via `dalc keys add`")
+			fmt.Println("Currently referencing this key using \"dalc\", but this can be changed in the config under the BlockSubmitter section")
+
 			return nil
 		},
 	}
-	command.Flags().String(pathFlag, config.DefaultConfigPath, "specific the home path")
+	command.Flags().String(homeFlag, config.HomeDir, "specific the home path")
 	return command
 }
 
 func startCmd() *cobra.Command {
-	const pathFlag = "path"
+	const homeFlag = "home"
 	command := &cobra.Command{
 		Use: "start",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			// load the config
-			path, err := cmd.Flags().GetString(pathFlag)
+			home, err := cmd.Flags().GetString(homeFlag)
 			if err != nil {
 				return err
 			}
 
-			cfg, err := config.Load(path)
+			dalcHome := config.ConfigPath(home)
+
+			cfg, err := config.Load(dalcHome)
 			if err != nil {
 				return err
 			}
 
 			// create the grpc server
-			srv, err := server.New(cfg)
+			srv, err := server.New(cfg, home, home+"/.celestia-light")
```

Comment:
This doesn't look cross-platform. Should `./celestia-light` also not be a constant?